tests/unittests/multimodal/ingest/ingest_tests.py

```
@@ -131,3 +132,52 @@ def test_does_not_read_directory_as_file(self, adapter, storage):
         adapter.get_scene_inventory.assert_has_calls(
             [call("/path/to/file"), call("/path/to/another_file")]
         )
+
+
+class SampleMatcher:
```

Ruff (0.15.12)

[warning] Object does not implement `__hash__` method

(PLW1641)